bip-0321.mediawiki

```diff
@@ -120,6 +120,12 @@ For payments completed on-chain, this is largely addressed by having the payment
 
 In order to avoid inadvertently revealing the sender's IP address or other information to the recipient, proof URIs must only be opened when they will simply switch to another locally-installed application (i.e. the application which initiated the payment). When clicking a URI from a website, the website should already have plenty of logic on its backend to process payment completion and a proof-of-payment callback is unnecessary.
 
+==Reference Implementation==
```

Comment:
Consistency nit:


```suggestion
== Reference Implementation ==
```

Reply:
I those this version because it is the most-used one in this repository (though not by a lot).